README.md

```diff
@@ -1,163 +1,178 @@
 # Four Keys
 
-
 # Background
 
-Through six years of research, the [DevOps Research and Assessment (DORA)](https://cloud.google.com/blog/products/devops-sre/the-2019-accelerate-state-of-devops-elite-performance-productivity-and-scaling) team has identified four key metrics that indicate the performance of a software development team.  This project allows you to collect your data and compiles it into a dashboard displaying these key metrics. 
-
+Through six years of research, the [DevOps Research and Assessment (DORA)](https://cloud.google.com/blog/products/devops-sre/the-2019-accelerate-state-of-devops-elite-performance-productivity-and-scaling) team has identified four key metrics that indicate the performance of a software development team. Four Keys allows you to collect data from your GitHub or GitLab project and compiles it into a dashboard displaying these key metrics.
 
+These four key metrics are:
 
 *   **Deployment Frequency**
 *   **Lead Time for Changes**
 *   **Time to Restore Services**
 *   **Change Failure Rate**
 
+# Who should use Four Keys
 
-# How it works
-
+Use Four Keys if:
 
+*   You want to measure your team's software delivery performance. For example, you may want to track the impact of new tooling or more automated test coverage, or you may want a baseline of your team's performance. 
+*   You have a project in GitHub or GitLab.
+*   Your project has deployments.
 
-1.  Events are sent to a webhook target hosted on Cloud Run
-1.  The Cloud Run target publishes all events to Pub/Sub
-1.  A Cloud Run instance is subscribed to the topics and does some light data transformation and inputs the data into BigQuery
-1.  Nightly scripts are scheduled in BigQuery to complete the data transformations and feed into the dashboard
+Four Keys does not work well with projects that have software releases because of how GitHub and GitLab collects data about releases. 
 
+For a quick baseline of your team's software delivery performance, use can also use the [DORA DevOps Quick Check](https://www.devops-research.com/quickcheck.html). The quick check also suggests DevOps capabilities you can work on to improve your performance. The Four Keys project itself can help you improve the following DevOps capabilities:
 
-![FourKeys Design](images/fourkeys-design.png)
+*   [Monitoring and observability](https://cloud.google.com/solutions/devops/devops-measurement-monitoring-and-observability)
+*   [Monitoring systems to inform business decisions](https://cloud.google.com/solutions/devops/devops-measurement-monitoring-systems)
+*   [Visual management capabilities](https://cloud.google.com/solutions/devops/devops-measurement-visual-management)
 
+# How it works
 
-# Code Structure
+1.  Events are sent to a webhook target hosted on Cloud Run. Events are any occurance in GitHub or GitLab that can be measured, such as a pull request or new issue. Four Keys defines events to measure, and you can add others that are relevant to your project.
```

Comment:
"any occurrence in your development environment (eg GitHub, GitLab, Cloud Build) that can be measured . . . "  

GitHub & GitLab are two current integrations (as is Cloud Build) - I don't want people to think events can only come from these two sources.

Reply:
Should we change the intro as well, so that it's broader than GitHub/GitLab?

"Four Keys allows you to collect data from your GitHub or GitLab project and compiles it into a dashboard displaying these key metrics."

could say:

"Four Keys allows you to collect data from your development environment (such as GitHub, GitLab, or Cloud Build) and compiles it into a dashboard displaying these key metrics."